moment/moment-node.d.ts

```diff
@@ -317,9 +332,12 @@ declare module moment {
         set(unit: string, value: number): Moment;
         set(objectLiteral: MomentInput): Moment;
         
-        /*This returns an object containing year, month, day-of-month, hour, minute, seconds, milliseconds.*/
-        //Works with version 2.10.5+
+        /* This returns an object containing year, month, day-of-month, hour, minute, seconds, milliseconds. */
+        // Works with version 2.10.5+
```

Comment:
make this one multiline comment block and use `@since`: http://usejsdoc.org/tags-since.html